.github/workflows/ci-web.yml

```diff
@@ -62,12 +62,23 @@ jobs:
         timeout-minutes: 5
 
       - name: Install web bindings
-        # Currently disable because `E2E tests & Unit tests` are.
-        if: false && steps.should-run-web-jobs.outputs.run == 'true'
-        run: npm run build:ci
+        if: steps.should-run-web-jobs.outputs.run == 'true'
+        run: |
+          # Execute 'npm clean-install' until success,
+          # This is done that way because sometime some CDN response with 503
```

Comment:
```suggestion
          # Execute 'npm clean-install' until success (or timeout),
          # Done this way because sometimes a CDN responds with 503
```